tests/conftest.py

```diff
@@ -10,7 +10,9 @@
 
 DEFAULT_SERVER = "http://localhost:8888/v1"
 DEFAULT_AUTH = "user:pass"
-DEFAULT_BUCKET = "default"
+DEFAULT_EDITOR_AUTH = "editor:pass"
+DEFAULT_REVIEWER_AUTH = "reviewer:pass"
+DEFAULT_BUCKET = "source"
 DEFAULT_COLLECTION = "source"
```

Comment:
I think we could make these tests closer to our current Remote Settings configuration. We could use `bucket=main-workspace` and `collection=product-integrity`.

The idea is to be able (eventually) to run this test suite against a freshly deployed instance.  But we can work towards that and adjust a few things in a follow-up PR